__tests__/app.test.js

```diff
@@ -106,6 +114,32 @@ describe("api/articles", () => {
         });
     });
   });
+  describe("GET api/articles", () => {
+    test("Status 200: responds with an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.length).toBe(13);
+          expect(body).toBeSortedBy("created_at");
```

Comment:
We should be checking that the array is sorted in descending order. If you take a look at the first example code block here, it shows how we can do this 🙂 
https://github.com/P-Copley/jest-sorted